### PR TITLE
Update setup.py for django 1.10

### PR DESCRIPTION
**Pull request by zavatskiy**

### CODE REVIEW

**Comment by mikemcgowan** [comment]
Is there a reason this hasn't been merged yet?

At the time of writing, when installing Django-CMS with Django version 1.10 using the [Django CMS installer](https://djangocms-installer.readthedocs.io/en/latest/index.html), it generates a `requirements.txt` file that can't be successfully compiled by [pip-tools](https://github.com/jazzband/pip-tools) because this plugin still requires `Django<1.10` ...

In other words, the Django CMS installer produces a `requirements.txt` that has requirements that can't actually be met ...

**Comment by FinalAngel** [comment]
@mikemcgowan probably just got forgotten, @yakky would you check and merge it? :)
